app/themes/user-admin/e2e/user-create/user-create.e2e-spec.ts

```diff
@@ -0,0 +1,77 @@
+import { UserAdminPage } from './user-create.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('User Create', function() {
+    let page: UserAdminPage;
+
+    beforeEach(() => {
+        page = new UserAdminPage();
+    });
+
+    it('should login successfully with right email and password', () => {
```

Comment:
You can move this entire function to a shared Page Object for abstraction. You can then call this PO function from all of your tests.